modules/emulator/R/GaussProcess.R

```diff
@@ -2,20 +2,21 @@
 ##' @title GaussProcess
 ##' @export
 ##'
-##' @param x
-##' @param y
-##' @param isotropic
+##' @param x Name of variable to plot on X axis
+##' @param y Name of variable to plot on Y axis
```

Comment:
I don't think this function plots anything. Y is our dependent variable (I assume a vector) and X is our set of independent variables (I assume a matrix?)